packages/ui/src/components/atoms/Logo/styles.scss

```diff
@@ -1,12 +1,11 @@
 [data-fs-logo] {
-  --fs-logo-width      : 7rem;
-  --fs-logo-img        : url("~/public/brandless-neutral.png");
+  --fs-logo-width : 7rem;
 
-  display: block;
   width: var(--fs-logo-width);
-  height: 100%;
+  padding: var(--fs-spacing-2) 0;
```

Comment:
Need to test this with different types of logos (more vertical, just an icon...)